maven-plugin-report-plugin/pom.xml

```diff
@@ -240,6 +237,7 @@
                                 <maven.compiler.source>${maven.compiler.source}</maven.compiler.source>
                                 <maven.compiler.target>${maven.compiler.target}</maven.compiler.target>
                             </properties>
+                            <junitPackageName>maven.invoker.it.plugin.report</junitPackageName>
```

Comment:
That looks unrelated. What is is used for ?

Reply:
Ok, not related.

It is used for package name of IT tests like 
https://ci-maven.apache.org/job/Maven/job/maven-box/job/maven-plugin-tools/job/master/lastCompletedBuild/testReport/maven.invoker.it/

Reply:
removed